### ISSUE TITLE
No eval_scoring.py file in the repo

### ISSUE THREAD

**Issue by cuongtran-uva** [neelsjain/NEFTune#12]
When I look at the eval_generate.py, I need eval_scoring.py to evaluate the performance. Can you upload this missing file?

**Comment by YuxinWenRick** [comment]
Hi, eval_scoring.py is not required. After generation, you can simply use alpaca_eval for evaluation, as shown here: https://github.com/neelsjain/NEFTune/blob/main/experiment_code/scripts/alpaca_eval.sh#L7.

Let us know if you have further questions!

**Comment by cuongtran-uva** [comment]
Thanks for your quick answer. I am trying to reproduce the experiments on a Jupyter notebook with 4 GPUs A100. The notebook is on our school's server. 
Whenever we set up the environment variables as in setup(rank, world_size, port) with world_size >1, the notebook hangs forever. It is fine when we set world_size = 1 but I guess we can not exploit data parallelism. 
We set up the conda environment properly as mentioned. 

1. Did you run successfully training models  (train.py )  on  a Jupyter notebook? Or you just submit train.py on a cluster of GPUs ?
